### PR TITLE
Wrap up blog post

### PR DESCRIPTION
**Pull request by rgaiacs**
This is the blog post to be posted on http://google-opensource.blogspot.com.br/.

@gvwilson @wking @pbanaszkiewicz @acroy @amitjamadagni @insertinterestingnamehere @izaid @mwiebe Could you take a few minutes and review this? I want to send it to Google until September 14th.

### CODE REVIEW

File: 2015/wrap-up-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+that supports and promotes world-class, innovative, open source scientific software
+(specially the scientific Python stack, [Julia](http://julialang.org/) and [rOpenSci](http://ropensci.org/)).
+
+This is the first time that NumFOCUS participated in Google Summer of Code (GSoC)
+although many of the projects that NumFOCUS support participated in many
+previous edition as mentoring organization or under the umbrella of one
+mentoring organization (for example the [Python Software
+Foundation](https://www.python.org/psf/)).
+
+The students working with us completed tree projects
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: three

**Comment by rgaiacs** [comment]
@moorepants Thanks.

File: 2015/wrap-up-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+[NumFOCUS](http://numfocus.org/)
+is a public charity in the United States
+that supports and promotes world-class, innovative, open source scientific software
+(specially the scientific Python stack, [Julia](http://julialang.org/) and [rOpenSci](http://ropensci.org/)).
+
+This is the first time that NumFOCUS participated in Google Summer of Code (GSoC)
+although many of the projects that NumFOCUS support participated in many
+previous edition as mentoring organization or under the umbrella of one
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: edition > editions
organization > organizations

File: 2015/wrap-up-blog-post.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The students working with us completed three projects
+and we are grateful for their incredible work,
+the many hours that their mentors spent during the summer
+and the amazing people that helped us with the GSoC application.
+
+#   Cython API for DyND
+
+[DyND](https://github.com/libdynd/libdynd) is a C++ library for dynamic, multidimensional arrays.
+The motivation for this project was make DyND available
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: was to make

File: 2015/wrap-up-blog-post.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+for the Python scientific stack,
+that uses Cython extensively,
+without having to go through the trouble of manually looping through each portion of an array.
+
+If you have at least once try to use Cython you know that this was not a easy
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: If you have ever used Cython ...

File: 2015/wrap-up-blog-post.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+project to developed. We are happy that Ian completed the project
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: developed > develop

File: 2015/wrap-up-blog-post.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+even he spent many hours hunting bugs and reorganizing code
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: even though he

Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: hunting for bugs

File: 2015/wrap-up-blog-post.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+(something that he did not expected).
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: expected > expect

File: 2015/wrap-up-blog-post.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+Read more about it at [Ian's post](https://insertinterestingnamehere.github.io/posts/gsoc-concluding-thoughts.html).
+
+#   Enhance AMY, a workshop-management platform for Software Carpentry
+
+The [Software Carpentry Foundation](http://software-carpentry.org/scf/index.html) (SCF)
+is a non-profit volunteer organization
+whose members teach researchers basic software skills.
+SCF started to run over a hundred workshops worldwide a year
+and manage the workshops became a problem.
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: manage > managing

File: 2015/wrap-up-blog-post.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+During the summer, Piotr worked to enhance [AMY](https://github.com/swcarpentry/amy/),
+a Django application that manage SCF's workshops,
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: manage > manages

File: 2015/wrap-up-blog-post.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+adding many new features, fixing bugs
+and helping SCF's program coordinators to keep updated
+with all the changes in AMY.
+
+The option for create yet another tool from the scratch
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: The option to create yet another tool from scratch

File: 2015/wrap-up-blog-post.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+instead of use an CRM solution like [CiviCRM](https://civicrm.org/)
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: an > a

File: 2015/wrap-up-blog-post.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+Read more about it at [Piotr's post](http://piotr.banaszkiewicz.org/blog/2015/08/15/amy-update-6/).
+
+#   JuliaQuantum: Framework for solvers
+
+[Julia](http://julialang.org/) is a high-level,
+high-performance dynamic programming language for technical computing,
+with syntax that is familiar to users of other technical computing environments.
+In the last few years Julia got many third party libraries
+and one of those are [Julia Quantum](https://juliaquantum.github.io/).
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: are > is

File: 2015/wrap-up-blog-post.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+During the summer Amit worked
+to integrate Quantum Monte-Carlo Wave Function Method as one of the solvers
```
Comment (moorepants) [inline 2015/wrap-up-blog-post.md]: integrate the Quantum

File: 2015/wrap-up-blog-post.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+The option to create yet another tool from the scratch
+instead of use a CRM solution like [CiviCRM](https://civicrm.org/)
+was based on the fact that (1) most of the SCF members have some knowledge of
+Python so they could help maintain AMY in the long run
+and (2) an small Django application could fit better for
+others organizations running software training like
+[PyLadies](http://www.pyladies.com/) and [Django Girls](http://www.pyladies.com/).
+
+Read more about it at [Piotr's post](http://piotr.banaszkiewicz.org/blog/2015/08/15/amy-update-6/).
```
Comment (pbanaszkiewicz) [inline 2015/wrap-up-blog-post.md]: AMY part looks okay. I'll have a newer post out today under this link:
http://piotr.banaszkiewicz.org/blog/2015/09/05/amy-update-7/

File: 2015/wrap-up-blog-post.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+[PyLadies](http://www.pyladies.com/) and [Django Girls](http://www.pyladies.com/).
+
+Read more about it at [Piotr's post](http://piotr.banaszkiewicz.org/blog/2015/09/05/amy-update-7/).
+
+#   JuliaQuantum: Framework for solvers
+
+[Julia](http://julialang.org/) is a high-level,
+high-performance dynamic programming language for technical computing,
+with syntax that is familiar to users of other technical computing environments.
+In the last few years Julia got many third party libraries
```
Comment (acroy) [inline 2015/wrap-up-blog-post.md]: I would suggest: "In the last few years Julia got many third party libraries, which integrate external libraries or use native implementations in Julia. One of those efforts is [JuliaQuantum](https://juliaquantum.github.io/), which aims to provide tools and frameworks for dealing with problems from quantum mechanics and quantum information science."

File: 2015/wrap-up-blog-post.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+and one of those is [Julia Quantum](https://juliaquantum.github.io/).
+
+During the summer Amit worked
```
Comment (acroy) [inline 2015/wrap-up-blog-post.md]: Maybe: "During the summer Amit worked on a framework for solving dynamical equations for JuliaQuantum. He integrated several solvers like the Quantum Monte-Carlo Wave Function Method. The [new interface](https://github.com/JuliaQuantum/QuDynamics.jl) makes it easy to add new solvers and to test different methods for a given problem."

**Pull request merged**